[Flutter] Non EVM support docs (#134)
reown-com/reown-docs: docs/appkit/flutter/core/siwe.mdx

WalletConnect uses permissions expressed as ReCaps to enable a One-Click Authentication.

## Pre-requisites

> In order for 1-CA + SIWE to work, you need to have a backend to communicate with. This backend will be used to generate a nonce, verify messages and handle sessions.

Reviewer: ```suggestion
For 1-CA + SIWE to function properly, a backend for communication is required. This backend will be used to generate a nonce, verify messages and handle sessions.
```

More info [here](https://docs.login.xyz/sign-in-with-ethereum/quickstart-guide/implement-the-backend)

:::info note

> This feature is compatible only with EVM blockchains, so if you decide to included non-EVM blockchains 1-CA + SIWE mechanism is going to be disabled internally.

Reviewer: ```suggestion
This feature is compatible only with EVM blockchains. Therefore, including non-EVM blockchains will result in the internal disabling of the 1-CA + SIWE mechanism.
```

I prefer not to use the word you or try to find another way
